gen/dcos-config.yaml

```
@@ -1826,6 +1842,12 @@ package:
               {
                   "Command": ["iptables-save"]
               },
+              {
+                  "Command": ["ip6tables-save"]
+              },
+              {
+                  "Command": ["ipset list"]
```

`ipset` is not installed on our CI.
Please add a test for it

---

It's installed, but there is a mistake in my code 🤦‍♂️

```suggestion
                  "Command": ["ipset", "list"]
```